test/test_distributions.py

```diff
@@ -259,6 +260,18 @@ def is_all_nan(tensor):
             'scale': torch.tensor([1e-5, 1e-5], requires_grad=True),
         },
     ]),
+    Example(LowRankMultivariateNormal, [
+        {
+            'loc': torch.randn(5, 2, requires_grad=True),
+            'scale_factor': torch.randn(5, 2, 1, requires_grad=True),
+            'scale_diag': torch.tensor([2.0, 0.25], requires_grad=True),
+        },
+        {
+            'loc': torch.randn(2, 3, requires_grad=True),
```

Comment:
nit: Could you make this `4,3` instead of `2,3` to make it clearer that `2` is the rank and `4` is the batch shape?

Reply:
yup